zebrad/src/application.rs

```diff
@@ -202,11 +203,16 @@ impl Application for ZebradApp {
 
         // Load config *after* framework components so that we can
         // report an error to the terminal if it occurs.
-        let config = command
-            .config_path()
-            .map(|path| self.load_config(&path))
-            .transpose()?
-            .unwrap_or_default();
+        let config = match command.config_path() {
+            Some(path) => match self.load_config(&path) {
+                Ok(config) => config,
+                Err(e) => {
+                    status_err!("Zebra could not parse the provided config file. This might mean you are using a deprecated format of the file.");
+                    return Err(e);
+                }
+            },
+            None => ZebradConfig::default(),
```

Comment:
Tracking is not active here yet, so I couldn't use `info!`. I wanted to say that `No config file provided. Using default options.` but left it out because I only found the `status_ok!` macro provided by Abscissa that uses `stdout`, so it would start sticking out once we redirect the output to a regular log file.

Reply:
Can we suggest how to fix the config in the `status_err!()` message?

Reply:
Done in [893c85b](https://github.com/ZcashFoundation/zebra/pull/4689/commits/893c85b925ef687722c2b860e085f9a409417eab).

Reply:
Now I'm not sure if I should add any newline characters to the message since it's long.

Reply:
I think either way is fine.